test/test_CellArrays2D.jl

```diff
@@ -1,27 +1,27 @@
-using Test, StaticArrays, AllocCheck, JustRelax
+using Test, Suppressor, StaticArrays, AllocCheck, JustRelax
 using ParallelStencil
```

Comment:
I believe there is no need to load `ParallelStencil` here

Reply:
There is, tests fail due to PS not being initialised.